scripts/review-loop.sh

```
@@ -0,0 +1,74 @@
+  local num="$1"
+  echo "===== PR #$num ====="
+  gh api "repos/$REPO/pulls/$num/reviews?per_page=100" --jq '.[] | select(.user.login|contains("coderabbitai") or contains("qodo")) | "\n--- \(.state) \(.submitted_at) ---\n\(.body[0:4000])"' 2>/dev/null | head -120
+}
+
+trigger_review() {
+  local num=$1
+  body="${2:-}"
+  if [ -n "$body" ]; then
+    gh pr comment "$num" --body "$body" >/dev/null 2>&1 && echo "triggered #$num"
+  else
+    gh pr comment "$num" --body "@coderabbitai full review" >/dev/null 2>&1 | echo "triggered #$num"
```

Shellcheck (0.11.0)

[warning] Piping to 'echo', a command that doesn't read stdin. Did you want 'cat' instead?

(SC2216)